content/en/docs/setup/install/istioctl/index.md

```diff
@@ -309,7 +309,7 @@ See [Customizing the installation configuration](/docs/setup/additional-setup/cu
 To completely uninstall Istio from a cluster, run the following command:
 
 {{< text bash >}}
-$ istioctl x uninstall --purge
+$ istioctl uninstall -y --purge
```

Comment:
Why are you changing the doc (adding -y)? This PR is just supposed to be promoting uninstall (remove x), so I don't think it should make any arbitrary other changes. For somebody learning about istioctl commands, which is what this doc is about, it's better to not suppress the prompt.
```suggestion
$ istioctl uninstall --purge
```